src/components/UI/TextField.tsx

```diff
@@ -76,9 +76,9 @@ const TextField: React.FC<TextFieldProps> = ({
   style = { color: colors.darkGray, borderColor: hasError ? colors.red : colors.darkGray, ...style };
 
   const leftIconStyle = { ...styles.icon, color: colors.darkGray, ...iconSyle };
-  let rightIconColor = !hasError ? colors.green : colors.red
+  let rightIconColor = !hasError ? colors.green : colors.red;
```

Comment:
`  let rightIconColor = hasError ? colors.red : colors.green;` could be easier to read